### DISCUSSION TITLE
integrate with vanta

### DISCUSSION THREAD

**Discussion by szamanr** [renovatebot/renovate#23644] [Suggest an Idea]
### Tell us more.

we've been very happy with renovate, but we're now using a tool called [vanta](https://www.vanta.com/) for SOC2 compliance. dependabot integrates with it nicely, so if we mark the vulerabilities as fixed in dependabot, they automatically get marked as completed in vanta.

therefore we'll be switching to dependabot for the time being. would be great if renovate integrated with vanta so that we can switch back.

**Comment by rarkins** [comment]
What do you mean by "mark vulnerabilities as fixed"? I'm not sure that applies to what Renovate does

**Reply by szamanr** [reply]
ah maybe you're right. dependabot gives you alerts when it detects a vulnerability. then it can open a PR fixing it, or you can manually mark is as done in a PR, or dismiss it. i guess that's not something that renovate does.

**Reply by rarkins** [reply]
You can still use renovate while using dependabot alerts

**Reply by HonkingGoose** [reply]
@szamanr Please read the [Renovate docs, `vulnerabilityAlerts`](https://docs.renovatebot.com/configuration-options/#vulnerabilityalerts) to learn how to use Dependabot alerts with Renovate.

**Reply by szamanr** [reply]
thanks! if renovate is configured to open PRs for vulnerabilities, and the PR gets merged, does it automatically resolve the alert?

also, is it possible for vulnerabilities PR to "skip the line" and not count towards the concurrently open PR limit?

**Reply by rarkins** [reply]
Yes, and yes

**Reply by HonkingGoose** [reply]
> also, is it possible for vulnerabilities PR to "skip the line" and not count towards the concurrently open PR limit?

I think more users will want to know this, so I'm putting it in the docs:

- #23778
